cpg_workflows/query_modules/seqr_loader.py

```diff
@@ -88,14 +83,9 @@ def annotate_cohort(
 
     logging.info('Annotating with clinvar and munging annotation fields')
     mt = mt.annotate_rows(
-        AC=mt.info.AC[mt.a_index - 1],
-        AF=mt.info.AF[mt.a_index - 1],
+        AC=mt.info.AC[0],  # TODO not sure about these two
```

Comment:
Do these need to go up into where the split happens now?

Reply:
I need to check the behaviour here, the splitting should mean these are either single numbers, or lists of length 1